src/t8_forest.h

```diff
@@ -929,6 +929,34 @@ void                t8_forest_element_coordinate (t8_forest_t forest,
                                                   int corner_number,
                                                   double *coordinates);
 
+/** Compute the coordinates of a point inside an element inside a tree.
```

Comment:
This function declaration should be moved to `t8_forest_geometrical.h`.

Reply:
Done